### ISSUE TITLE
Adding a hover text

### ISSUE THREAD

**Issue by Tanisha257** [GDG-On-Campus-BVM/GDG-On-Campus-BVM-Website#3]
Under the pictures of the team member, adding a hover text on the symbols of ig, github and linkedin might make it look better.

**Comment by JaiminPatel345** [comment]
mobile me hover effect kese add kr skte he ?
hover means when mouse/cursor came over any section 

do you taking about add text when click ?

**Comment by Tanisha257** [comment]
Yes the chota sa hover text ata hai when ur arrow is on the link...that one jese ke this![image](https://github.com/user-attachments/assets/71ad0458-4a0e-40a1-a316-04c000bdee3b)

**Comment by JaiminPatel345** [comment]
okk i thought you taking about mobile 

done
![image](https://github.com/user-attachments/assets/fe8ff74e-2dac-4280-9a5a-76eaeb329ae2)
